app/konnect/mesh-manager/service-mesh.md

```diff
@@ -23,6 +23,13 @@ Creating a fully-functioning {{site.mesh_product_name}} deployment in {{site.kon
 1. Click **New Control Plane** and complete the fields as needed.
 1. Select the control plane you just created, and then click **Zones** in the sidebar.
 1. Click **Create Zone**, configure the fields as needed, and follow the steps in the wizard to connect your zone to {{site.konnect_short_name}}.
+   
+     {:.note}
+    > **Note:** Mesh Manager automatically creates a [managed service account](/konnect/org-management/system-accounts/) that is only used in the zone creation process. 
```

Comment:
I'd say it's used to issue a token during the zone creation process. Because in practice this account is how zones auth

Reply:
+1, I thought I added a comment like this as well but I can't find it now :facepalm:

Reply:
Fixed this in the system accounts page.